cpp/ql/src/semmle/code/cpp/models/implementations/Memset.qll

```diff
@@ -15,7 +15,7 @@ import semmle.code.cpp.models.interfaces.SideEffect
 private class MemsetFunction extends ArrayFunction, DataFlowFunction, AliasFunction,
   SideEffectFunction {
   MemsetFunction() {
-    hasGlobalName(["memset", "wmemset", "bzero", "__builtin_memset", "__builtin_memset_chk"]) or
+    hasGlobalName(["memset", "wmemset", bzero(), "__builtin_memset", "__builtin_memset_chk"]) or
```

Comment:
`bzero` can also live in the `std` namespace, I think? As can `memset`, so I think this charpred should be refactored.

Reply:
Indeed. I've only refactored the `strcpy` class wrt. this. I can do it for the other ones as well, though.